packages/cli/src/commands/target/index.ts

```diff
@@ -0,0 +1,55 @@
+import Client from '../../util/client';
+import getArgs from '../../util/get-args';
+import getInvalidSubcommand from '../../util/get-invalid-subcommand';
+import { help } from '../help';
+import list from './list';
+import { targetCommand } from './command';
+import { getLinkedProject } from '../../util/projects/link';
+
+const COMMAND_CONFIG = {
+  ls: ['ls', 'list'],
+};
+
+export default async function main(client: Client) {
+  let subcommand: string | string[];
+
+  const argv = getArgs(client.argv.slice(2), {
+    '--next': Number,
+    '-N': '--next',
+  });
+
+  if (argv['--help']) {
+    client.output.print(
+      help(targetCommand, { columns: client.stderr.columns })
+    );
+    return 2;
+  }
+
+  argv._ = argv._.slice(1);
+  subcommand = argv._[0] || 'list';
+  const args = argv._.slice(1);
+  const { cwd, output } = client;
+  const link = await getLinkedProject(client, cwd);
+
+  if (link.status === 'error') {
+    return link.exitCode;
+  }
+
+  if (link.status === 'not_linked') {
+    // TODO: prompt for link
```

Comment:
Non-blocking: Do we still want to come back to this, or is this just for future reference?

Reply:
Made a card for this feature.